From 0fec1c1f5d0eca95995fdd8527a60e73d18d59c4 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:06:57 -0400
Subject: [usb] uas: Remove comment about registering a uas scsi controller for each usb bus

Message-id: <1410887463-20674-346-git-send-email-dzickus@redhat.com>
Patchwork-id: 95376
O-Subject: [RHEL7 PATCH 345/591] uas: Remove comment about registering a uas scsi controller for each usb bus
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 7cace978fba5d0ec6eed50509cda40eea85f8e98
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Tue Oct 29 11:36:43 2013 +0100

    uas: Remove comment about registering a uas scsi controller for each usb bus

    Although an interesting concept, I don't think that this is a good idea:

    -This will result in lots of "virtual" scsi controllers confusing users
    -If we get a scsi-bus-reset we will now need to do a usb-device-reset of all
     uas devices on the same usb bus, which is something to avoid if possible

    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/storage/uas.c b/drivers/usb/storage/uas.c
index 08e9710..a7ac97c 100644
--- a/drivers/usb/storage/uas.c
+++ b/drivers/usb/storage/uas.c
@@ -1050,12 +1050,6 @@ static void uas_free_streams(struct uas_dev_info *devinfo)
  usb_free_streams(devinfo->intf, eps, 3, GFP_KERNEL);
 }
 
-/*
- * XXX: What I'd like to do here is register a SCSI host for each USB host in
- * the system.  Follow usb-storage's design of registering a SCSI host for
- * each USB device for the moment.  Can implement this by walking up the
- * USB hierarchy until we find a USB host.
- */
 static int uas_probe(struct usb_interface *intf, const struct usb_device_id *id)
 {
  int result = -ENOMEM;
-- 
1.7.1